static-site/src/sections/individual-group-page.jsx

```diff
@@ -0,0 +1,211 @@
+        if (sourceInfo && (sourceInfo.showDatasets || sourceInfo.showNarratives)) {
+          return fetchAndParseJSON(`/charon/getAvailable?prefix=/groups/${groupName}/`);
+        }
+        return undefined;
+      })
+      .then((availableDataJson) => {
+        if (availableDataJson) {
+          this.setState({
+            datasets: this.createDatasetListing(availableDataJson.datasets, groupName),
+            narratives: this.createDatasetListing(availableDataJson.narratives, groupName),
+            dataLoaded: true,
+            groupName
+          });
+        }
+      })
+      .catch((err) => {
+        console.error("Error fetching / parsing data.", err.message);
+        this.setState({errorFetchingData: true});
+      });
+  }
+
+  render() {
+    return (
+      <GenericPage location={this.props.location}>
+        {this.state.sourceInfo &&
+        <>
+          <FlexCenter>
+            <Title avatarSrc={this.state.sourceInfo.avatar}>
+              {this.state.sourceInfo.title}
+              <Byline>{this.state.sourceInfo.byline}</Byline>
+              <Website>{this.state.sourceInfo.website}</Website>
+            </Title>
+          </FlexCenter>
+          {this.state.sourceInfo.overview &&
+            <FlexCenter>
+              <OverviewContainer>
+                {this.state.sourceInfo.overview}
+              </OverviewContainer>
+            </FlexCenter>
+          }
+        </>}
+        <HugeSpacer />
+        {this.state.dataLoaded && this.state.sourceInfo && this.state.sourceInfo.showDatasets && this.state.datasets.length > 0 && (
+          <ScrollableAnchor id={"datasets"}>
+            <div>
+              <splashStyles.H3>Available datasets</splashStyles.H3>
+              <DatasetSelect
+                datasets={this.state.datasets}
+                columns={[
+                  {
+                    name: "Dataset",
+                    value: (dataset) => dataset.filename.replace(/_/g, ' / ').replace('.json', ''),
+                    url: (dataset) => dataset.url
+                  }
+                ]}
+              />
+            </div>
+          </ScrollableAnchor>
+        )}
+        <HugeSpacer />
+        {this.state.dataLoaded && this.state.sourceInfo && this.state.sourceInfo.showNarratives && this.state.narratives.length > 0 &&(
+          <ScrollableAnchor id={"narratives"}>
+            <div>
+              <splashStyles.H3>Available narratives</splashStyles.H3>
+              <DatasetSelect
+                datasets={this.state.narratives}
+                columns={[
+                  {
+                    name: "Narrative",
+                    value: (dataset) => dataset.filename.replace(/_/g, ' / ').replace('.json', ''),
+                    url: (dataset) => dataset.url
+                  }
+                ]}
+                unit="narrative"
+              />
+            </div>
+          </ScrollableAnchor>
+        )}
+        { this.state.errorFetchingData &&
```

Comment:
`errorFetchingData` is set if the `getAvailable` call fails. In this case, we should display the group page as normal, but swap out the `DatasetSelect` component(s) for an error message saying "Listing of datasets failed, please contact us at ..."